src/mobile-optimization/widget-expanded-listener.ts

```diff
@@ -0,0 +1,38 @@
+import * as FlexWebChat from '@twilio/flex-webchat-ui';
+
+export const WIDGET_EXPANDED_CLASS = 'AseloWidget-Expanded';
+
+/**
+ * This function adds/removes '.AseloWidget-Expanded' class to the widget's
+ * floating button. On a mobile device, this class will hide the floating button.
+ */
+function addOrRemoveWidgetExpandedClass(manager: FlexWebChat.Manager) {
+  const entryPointButton = document.querySelector<HTMLButtonElement>('button.Twilio-EntryPoint');
+  console.log({ manager, entryPointButton });
+  if (!entryPointButton) {
+    return;
+  }
+
+  const { isEntryPointExpanded } = manager.store.getState().flex.session;
+
+  if (isEntryPointExpanded) {
+    entryPointButton.classList.add(WIDGET_EXPANDED_CLASS);
+  } else {
+    entryPointButton.classList.remove(WIDGET_EXPANDED_CLASS);
+  }
```

Comment:
I'm not against doing this, but this does sounds like a good fit for a thunk? That way we can see the Redux state without relying on this `manager.store.getState` API that I always felt a bit hacky? Anyways, just throwing in an idea. Leaving this a separate comment as I'm actually not sure that's best. Ignore this comment for now :P